<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chose ticket</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>My tickets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Feedback</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>From: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose direction</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find ticket!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
